### ISSUE TITLE
Erro de acessibilidade: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### ISSUE THREAD

**Issue by Ramonsouzadasilva** [Ramonsouzadasilva/WeGIA#1]
**ID da Violação**: color-contrast
        **Descrição**: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
        **Ajuda**: Elements must meet minimum color contrast ratio thresholds

        **Detalhes**:
        - Impacto: serious
        - Nós afetados: 1

        **Recomendações para Correção**:
        
            **Elemento 1:**
            - **HTML**: `<input type="submit" value="Entrar" class="btn btn-primary hidden-xs entrar">`
            - **Alvo**: .btn
            - **Mensagem de Ajuda**: Fix any of the following:
  Element has insufficient color contrast of 3.89 (foreground color: #ffffff, background color: #0088cc, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
            - **Sugestão**: Verifique https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=axe-puppeteer para orientações detalhadas.
